### DISCUSSION TITLE
Should we update gitignore to ignore pnpm-lock file?

### DISCUSSION THREAD

**Discussion by mayank1513** [pmndrs/zustand#2014] [Polls]
It's good to have only one lock file. Some devs may be using different package managers. npm-lock is already ignored in the .gitignore. However, pnpm-lock.yaml is not ignored. No need to track this file.

**Comment by dai-shi** [comment]
Thanks for opening up a discussion! Yeah, it already ignores `package-lock.json`. So, to be consistent, removing it or adding `pnpm-lock.yaml` seems to make sense.
@sewera your thoughts?

**Reply by sewera** [reply]
Let's ignore pnpm

**Reply by dai-shi** [reply]
Thanks.
@mayank1513 Please open a new PR.
